### PR TITLE
Move pvc_snapshot_and_clone folder to workloads dir

### LINKED ISSUE #4781: test_pgsql_pvc_snapshot fails on teardown with Attribute error
Tests test_pgsql_pvc_snapshot  fails on teardown with Attribute error as shown below:

```
        # Delete created postgres and pgbench pods
        log.info("Deleting postgres pods which are attached to restored PVCs")
>       for pgsql_obj in self.pgsql_obj_list:
E       AttributeError: 'TestPvcSnapshotOfWorkloads' object has no attribute 'pgsql_obj_list'

tests/e2e/lifecycle/pvc_snapshot/test_pgsql_pvc_snapshot.py:25: AttributeError
```

### PR DESCRIPTION
**Pull request by Akarsha-rai**
- Move  pvc_snapshot_and_clone folder to workloads dir
- Fixes #4781

Signed-off-by: Akarsha-rai <akrai@redhat.com>

### CODE REVIEW

**Review by PrasadDesala — APPROVED** [review]

**Review by PrasadDesala — DISMISSED** [review]

File: ocs_ci/ocs/benchmark_operator.py [hunk 1]
```diff
@@ -18,14 +18,12 @@
...
+from ocs_ci.ocs.constants import BMO_NAME
 from ocs_ci.ocs.exceptions import CommandFailed
 from ocs_ci.ocs.node import get_worker_nodes
 from ocs_ci.ocs.ocp import OCP, switch_to_default_rook_cluster_project
 from ocs_ci.utility.utils import TimeoutSampler
 
-# BMO stand for : BenchMark Operator
-# The benchmark operator name used for path / namespace etc.
-BMO_NAME = "benchmark-operator"
 # The benchmark operator git repository
```
Comment (PrasadDesala) [inline ocs_ci/ocs/benchmark_operator.py:27]: Any specific reason for moving this to constants?  There are many places in perf/e2e tests using this path already.

Comment (Akarsha-rai) [reply ocs_ci/ocs/benchmark_operator.py]: Moved back to benchmark file

**Review by Avilir — APPROVED** [review]
LGTM

**Review by PrasadDesala — APPROVED** [review]

File: ocs_ci/ocs/resources/pvc.py [hunk 2]
```diff
@@ -448,6 +448,7 @@ def create_pvc_clone(
     sc_name,
     parent_pvc,
     clone_yaml,
+    namespace,
```
Comment (jilju) [inline ocs_ci/ocs/resources/pvc.py:451]: Looks good. Other tests which is using this function are updated.

**Review by jilju — APPROVED** [review]

**Pull request merged**